apps/protocol-api/src/prisma/schema.prisma

```diff
@@ -285,16 +301,18 @@ model ContributionStatus {
 // There should be a mapping
 // activity to user
 model ActivityType {
-  id               Int                    @id @default(autoincrement())
-  createdAt        DateTime               @default(now())
-  updatedAt        DateTime               @default(now()) @updatedAt
-  name             String                 @unique
-  active           Boolean                @default(true)
-  default          Boolean                @default(false)
-  users            UserActivity[]
-  contributions    Contribution[]
-  categoryActivity CategoryActivityType[]
-  guilds           GuildActivityType[]
+  id                Int                    @id @default(autoincrement())
+  createdAt         DateTime               @default(now())
+  updatedAt         DateTime               @default(now()) @updatedAt
+  name              String                 @unique
+  active            Boolean                @default(true)
+  default           Boolean                @default(false)
+  users             UserActivity[]
+  contributions     Contribution[]
+  categoryActivity  CategoryActivityType[]
+  guilds            GuildActivityType[]
+  GuildActivityType GuildActivityType[]
+  GuildActivityType GuildActivityType[]
```

Comment:
This duplication is causing an error in the build -- Should one of these be the `CanonicalGuildActivityType`?